models/model.js

```diff
@@ -156,3 +157,10 @@ exports.removeComment = (comment_id) => {
     return db.query(queryStr, [comment_id]);
   });
 };
+
+exports.fetchEndpointList = () => {
+  return readFile("./endpoints.json").then((endpoints) => {

```

Comment:
Great logic, but this should be in the controller as it isn't an interaction with the database. Move this across, and then it's good to merge 👍